README.md

```diff
@@ -58,4 +61,8 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
\ No newline at end of file
+This project is [MIT](./MIT.md) licensed.
+
+## Video Take Through Link
+
+https://www.loom.com/share/64e4743f620540e69489c0e37b8f1f3e
```

Comment:
- [ ] The README should contain an "Acknowledgements" which gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons](https://creativecommons.org/licenses/) License. Please consider adding it to your acknowledgment section of the README. 😄

![](https://i.imgur.com/7faFGWN.png)